NT-Line-team/Behave_test
# Khanhvnv week7: Develop test scenarios using features file for Opencart web application

## Exercise

* [Exercise-7](https://github.com/NT-Line-team/Behave_test/issues/9)

## Overview

Develop test scenarios using features file of below function:
```
Login
Register
Forgot password
Search a product
Buy a product and checkout
Add review for product

Refer below for publish site:
https://demo.opencart.com/index.php?route=common/home
```
## Verification Steps
git checkout pr/###
Open and verify the scenario for 6 features:
1. `behave -k .\features\login.feature
2. behave -k .\features\register.feature
3. behave -k .\features\search_product.feature
4. behave -k .\features\buy_checkout.feature
5. behave -k .\features\add_review.feature
6. behave -k .\features\forgot_password.feature
